docfiles/docs.html

```diff
@@ -21,12 +25,12 @@
 
     <!-- @include toc.html -->
     
-    <div class="article">
+    <div class="article" id="maincontent">
         
         <!-- @include header.html -->
 
         <div id="docs" class="main ui container mainbody">
-            <button id="printbtn" class="circular ui icon right floated button hideprint">
+            <button id="printbtn" class="circular ui icon right floated button hideprint" title="Print this page">
```

Comment:
are these strings picked up in "webstrings.json"? (we can fix that in a future PR if needed)